lib/file-view.coffee

```diff
@@ -19,7 +19,11 @@ class FileView extends HTMLElement
     @fileName.dataset.name = @file.name
     @fileName.dataset.path = @file.path
 
-    @fileName.classList.add(FileIcons.getService().iconClassForPath(@file.path))
+    iconClass = FileIcons.getService().iconClassForPath(@file.path, this)
+    if iconClass
+      unless Array.isArray iconClass
```

Comment:
In other words, `["first", "second"]` will end up being concatenated and added as a single class. E.g.:

```html
<element class="first,second">
```